mlrun/serving/states.py

```diff
@@ -843,9 +857,10 @@ def get_step_class_object(self, namespace):
                 class_object = get_class(class_name or self._default_class, namespace)
         return class_object, class_name
 
-    def _is_local_function(self, context):
+    def _is_local_function(self, context, current_function=None):
```

Comment:
to keep compatibility, consider add the opt `current_function` param also to the `BaseStep` class method